.circleci/config.yml

```diff
@@ -19,6 +19,10 @@ workflows:
             - test-node-v6
             - test-node-v8
             - docker-build
+      - electron-pack-and-release:
```

Comment:
This job didn't run:

![image](https://user-images.githubusercontent.com/6199391/38319396-962c0272-3829-11e8-9272-96909e17deba.png)

Reply:
@n-riesco It appears to me that "electron-pack-and-release" is running: https://circleci.com/gh/plotly/image-exporter/tree/test-builds-across-travis-circle-appveyor

(See #1124 for example.)

Please pardon, am I missing something simple?

Reply:
@JamesCropcho For some reason, github doesn't show the build (as seen on the above screenshot). Since circleCI runs the job, then this isn't an issue.